packages/emotion-primitives/src/css.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+import transform from 'css-to-react-native'
+import { StyleSheet } from 'react-primitives'
+import { interleave } from './utils'
+
+export function css(...args: any) {
+  let vals
+
+  if (args[0] == null || args[0].raw === undefined) {
+    vals = args
+  } else {
+    vals = interleave(args)
+  }
+
+  let styles = []
+  let buffer = ''
+  let lastType
+
+  let handleInterpolation = (interpolation: *, i: number, arr: Array<*>) => {
```

Comment:
`css` being possibly a hot path it would be good to avoid closure here, right?

Reply:
after a thought im not sure if this creates a closure 🤔 kinda blur line for me sometimes between closure and scope lookup, anyway - as it being hot it would be good to have single copy of this function anyway